ros/src/tl_detector/CMakeLists.txt

```diff
@@ -200,4 +200,8 @@ include_directories(
 # endif()
 
 ## Add folders to be run by python nosetests
-# catkin_add_nosetests(test)
+#catkin_add_nosetests(test)
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+  add_rostest(test/test_tl_detector.launch)
+endif()
```

Comment:
why would github show this 'no newline at the end of the file' warning? could it affect anything in any language?

Reply:
I saw in the past some tools having some problems with it. According to the [stackoverflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) discussion, there are actually some implications with it.